### PR TITLE
Add more telemetry for sql database projects

### PR DESCRIPTION
**Pull request by kisantia**
Adding telemetry that was missing for a few scenarios: 
- create project from db
- update project from db
- add existing item
- target platform of new projects
- number of sqlcmd vars and project's target platform version when publishing
- publish to container

### CODE REVIEW

File: extensions/sql-database-projects/src/controllers/projectController.ts [hunk 1]
```diff
@@ -270,6 +274,8 @@ export class ProjectsController {
 	public async publishToDockerContainer(context: Project | dataworkspace.WorkspaceTreeItem, deployProfile: IDeployProfile): Promise<void> {
 		const project: Project = this.getProjectFromContext(context);
 		try {
+			TelemetryReporter.sendActionEvent(TelemetryViews.ProjectController, TelemetryActions.publishToContainer);
```
Comment (kisantia) [inline extensions/sql-database-projects/src/controllers/projectController.ts:277]: @llali I can remove the publish to container ones if you were planning on adding telemetry for it

**Review by Charles-Gagnon — APPROVED** [review]

File: extensions/sql-database-projects/src/common/telemetry.ts [hunk 2]
```diff
@@ -35,5 +36,8 @@ export enum TelemetryActions {
 	changePlatformType = 'changePlatformType',
 	updateSystemDatabaseReferencesInProjFile = 'updateSystemDatabaseReferencesInProjFile',
 	startAddSqlBinding = 'startAddSqlBinding',
-	finishAddSqlBinding = 'finishAddSqlBinding'
+	finishAddSqlBinding = 'finishAddSqlBinding',
```
Comment (udeeshagautam) [inline extensions/sql-database-projects/src/common/telemetry.ts:39]: Probably not in scope for this PR but for the start/finish telemetry points - are these to check if the operation was successful or for calculating perf?

File: extensions/sql-database-projects/src/controllers/projectController.ts (hunk 1 above)
Comment (llali) [reply extensions/sql-database-projects/src/controllers/projectController.ts:277]: it's up to you. If you remove it, I' send a PR to add it back. I' thinking to keep it if you already add it but if adding too much work for you (testing verifying ...) I can add it in mine

**Review by llali — APPROVED** [review]

**Review by zijchen — APPROVED** [review]

File: extensions/sql-database-projects/src/controllers/projectController.ts (hunk 1 above)
Comment (kisantia) [reply extensions/sql-database-projects/src/controllers/projectController.ts:277]: ok I'll keep it for now, and you can add to it later if there are additional things you wanna add

**Pull request merged**